docs/UserGuide.md

```diff
@@ -133,8 +149,14 @@ Examples:
 ### View student statistics: `view`
 This would display the following in a pop-up window 
 - the total number of students 
-- the total amount owed by students
-- the number of upcoming exams in the following month
+- the total amount owed by students (Currently shows the exact amount)
+- the number of upcoming exams in following 1 month period (from today up to the same day of the next month)
```

Comment:
More super minor nits!

`- the total amount owed by students (exact, will not be rounded to 2 decimal places)`
`- the number of upcoming exams in the following month (from today up to the same day of the next month)`